PC02_Graffiti_HEISNER.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: YOUR NAME
+May 29, 2020
+'''
+
+from turtle import * #import the library of commands that you'd like to use
+
+colormode(255)
+
+# Create a panel to draw on. 
+panel = Screen()
+w = 750 # width of panel
+h = 750 # height of panel
+panel.setup(width=w, height=h) #600 x 600 is a decent size to work on. 
+#You can experiment by making it the size of your screen or super tiny!
+
+# Create a colorful background and add Bezos image to it
+image = "Bezos.gif"
+panel.bgcolor("lightsteelblue1")
+panel.bgpic(image)
+
+#=======Add your code here======
+
```

Comment:
add definitions for variables (I used these in changes later on in the code)
```suggestion
longSide = 45 #sets the long side of the eyebrow
shortSide = 12 #sets the short side of the eyebrow
yellow = (250,200,0)
orange = (255,153,51)
triAngle = 110 #sets the first angle in the goatee triangle
triSide = 52 #sets the side length of the triangle
stickLength = 267 #sets the length of the stick
```